ocaml/xapi/xapi_cluster_host.mli

```diff
@@ -45,6 +45,11 @@ val create : __context:Context.t -> cluster:API.ref_Cluster -> host:API.ref_host
 (** [create ~__context ~cluster ~host] is implementation of the XenAPI call
     'Cluster_host.create'. It is the Cluster_host object constructor *)
 
+val force_destroy : __context:Context.t -> self:API.ref_Cluster_host -> unit
+(** [force_destroy ~__context ~self] is the implementation of the XenAPI call
+    'Cluster_host.force_destroy'. It forcefully removes the DB object and
```

Comment:
It seems that it does not always remove the DB object, only when it succeeded to destroy the cluster.